### ISSUE TITLE
new pruning selection technique

### ISSUE THREAD

**Issue by thushv89** [thushv89/AdaCNN#15]
filters/neurons with mean(abs(weights))<threshold will be removed

**Comment by thushv89** [comment]
Accuracy Drops for CIFAR-10 Stationary various pruning techniques
getting minimum from tf.reduce_mean(tf.abs(w)) : 44.87 -> 15.55
getting minimum from tf.reduce_mean(w) : 41.52 -> 9.99
removing the newest_weights : 42.22 -> 42.35
